frontend/apps/web/components/ui/dialog.tsx

```diff
@@ -12,14 +12,16 @@ const DialogTrigger = DialogPrimitive.Trigger;
 
```

Comment:
I pulled in the latest shadcn dialog to get the `<DialogClose />` component.

It included some updates to the styling.